exonum/src/encoding/spec.rs

```diff
@@ -14,7 +14,8 @@
 
 /// `encoding_struct!` macro implements a structure that can be saved in the Exonum blockchain.
 ///
-/// The macro creates getter methods for all fields with the same names as fields.
+/// The macro creates getter methods for all fields; these getter methods will
```

Comment:
The macro creates getter methods in respect of all its fields. The names of the methods coincide with the fields names.